quickwit-index/src/actors/uploader.rs

```diff
@@ -105,38 +97,26 @@ async fn put_split_files_to_storage(
     split_metadata: SplitMetadata,
     storage: &dyn Storage,
 ) -> anyhow::Result<Manifest> {
-    info!("upload-split");
+    let bundle_path = split.split_scratch_directory.path().join(BUNDLE_FILENAME);
+    info!("upload-split-bundle {:?} ", bundle_path);
     let start = Instant::now();
 
     let mut upload_res_futures = vec![];
-    let mut manifest_entries = Vec::new();
-    for (path, file_size_in_bytes) in &split.files_to_upload {
-        let file_name = path
-            .file_name()
-            .and_then(|filename| filename.to_str())
-            .map(|filename| filename.to_string())
-            .with_context(|| format!("Failed to extract filename from path {}", path.display()))?;
-        manifest_entries.push(ManifestEntry {
-            file_name: file_name.to_string(),
-            file_size_in_bytes: *file_size_in_bytes,
-        });
-        let key = PathBuf::from(file_name);
-        let payload =
-            quickwit_storage::PutPayload::from(split.split_scratch_directory.path().join(path));
-        let upload_res_future = async move {
-            storage.put(&key, payload).await.with_context(|| {
-                format!(
-                    "Failed uploading key {} in bucket {}",
-                    key.display(),
-                    storage.uri()
-                )
-            })?;
-            Result::<(), anyhow::Error>::Ok(())
-        };
-        upload_res_futures.push(upload_res_future);
-    }
+    let key = PathBuf::from(BUNDLE_FILENAME);
```

Comment:
Where do we pass the bundeoffsets?

Reply:
it's inside `SplitMetadata`, the split into `SplitMetadataAndBundleOffsets` can be done separately